lib/AppInfo/Application.php

```diff
@@ -29,30 +29,18 @@
 namespace OCA\Adminly_Dashboard\AppInfo;
 
 use OCP\AppFramework\App;
-use OCP\AppFramework\Bootstrap\IBootContext;
-use OCP\AppFramework\Bootstrap\IBootstrap;
-use OCP\AppFramework\Bootstrap\IRegistrationContext;
 
-class Application extends App implements IBootstrap
+class Application extends App
 {
-
     /** @var string */
-    public const APP_NAME = 'adminly_dashboard';
+    public const APP_ID = 'adminly_dashboard';
```

Comment:
I would prefer if the constant stays APP_NAME for consistency through the rest of the code, and how it's used.